### PR TITLE
DEPR: plotting deprecations

### PR DESCRIPTION
**Pull request by jbrockmendel**
Input/help needed on two outstanding questions:

1) Do we just remove `pandas.tseries.converter` after this?
2) Some of these deprecations are old enough that I couldn't track them down via the blame, so they currently have "(:issue:`????`)" in the whatsnew.

### CODE REVIEW

File: doc/source/whatsnew/v1.0.0.rst [hunk 1]
```diff
@@ -499,6 +499,10 @@ or ``matplotlib.Axes.plot``. See :ref:`plotting.formatters` for more.
 
 **Other removals**
 
+- Removed the previously deprecated :func:`pandas.plotting._matplotlib.tsplot`, use :meth:`Series.plot` instead (:issue:`19980`)
+- :func:`pandas.tseries.converter.register` has been moved to :func:`pandas.plotting.register_matplotlib_converters` (:issue:`18307`)
+- :meth:`Series.plot` no longer accepts positional arguments, pass keyword arguments instead (:issue:`????`)
```
Comment (jreback) [inline doc/source/whatsnew/v1.0.0.rst]: just use the PR number here instead

**Review by jreback — APPROVED** [review]
lgtm ex small change (for 2)

1) pandas.tseries.converter, yes let's remove this entirely

**Comment by jbrockmendel** [comment]
updated+green

**Comment by jreback** [comment]
thanks @jbrockmendel

**Pull request merged**